controllers/user_session.go

```diff
@@ -0,0 +1,72 @@
+package controllers
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/foolin/goview/supports/ginview"
+	"github.com/gin-gonic/gin"
+	"github.com/go-playground/validator/v10"
+	errorHandler "github.com/gutakk/go-google-scraper/helpers/error_handler"
+	session "github.com/gutakk/go-google-scraper/helpers/session"
+	"github.com/gutakk/go-google-scraper/models"
+)
+
+const (
+	loginTitle = "Login"
+	loginView  = "login"
```

Comment:
Not sure why you picked the file name `user_session.go` 🤔  It seems inspired by Rails and Devise `sessions_controller.rb` 😅  It's not consistent with `register.go`. Wouldn't `login.go` be more consistent? If you prefer to keep `session.go`, then `register.go` should be renamed to `registration.go` 💭

Reply:
Yes I thought about rails when I did this 😅. I decided to change from `user_session.go` to `login.go`
Fixed in 95fd1731700d3159f4f4167625f0383e186d0e43

Reply:
Allgood, that's what I thought also 😁